.travis.yml

```diff
@@ -18,6 +20,12 @@ script:
 # Dump trial log on failure.
 after_failure: "cat _trial_temp/test.log"
 
+# Only run test when committing to master branch.
+branches:
+  only:
+    - master
+    - \d+\.\d+\.\d+
```

Comment:
I think I'm going to land this immediately (it's a valid config change) and we can do the release fixing separately.